### DISCUSSION TITLE
Lib required (?) and EEPROM.h is missing

### DISCUSSION THREAD

**Discussion by mobil750** [earlephilhower/arduino-pico#1646] [General]
Hello, I use latest Adruino IDE 2.x and installed V3.3.2 of "Raspberry Pi Pico/RP2040" lib.

Do I need additionally the "Arduino Mbed OS RP2040 Boards" lib? It is claimed when I start compilation. I thought your lib would be sufficient. 

When installing the "Arduino Mbed OS RP2040 Boards" I get the the error message that EEPROM.h is missing. I see it is part of the "Raspberry Pi Pico/RP2040" lib according Github.
What am I doing wrong here?

Thanks in advance

Helmut

**Comment by earlephilhower** [comment]
I can't speak for why Arduino tries to install their generic RP2040 support, but I think all you need to do is make sure you are building using the correct board.

You've probably ended up using their core which doesn't support a lot of things, like EEPROM.  Just select this one instead and you should be okay.

**Reply by mobil750** [reply]
Thanks, that did it. 
But now I get some errors regarding "assertions" which I cannot interpret. Could you give me a hint what is missing in my coding or where I can read a doc regarding this issue. The compiler output is shown in the attachment 

[assertion_errors.rtf.zip](https://github.com/earlephilhower/arduino-pico/files/12346598/assertion_errors.rtf.zip)

**Reply by earlephilhower** [reply]
Looks like you need to check out the library you're using.  It's trying to do something incorrect, leading to the assertions.  Maybe you need to `#define` something or other?
